### PR TITLE
Pull Request for My Portfolio

### PR DESCRIPTION
**Pull request by DebbyMiressa**
This ('MyPortfolio') project is an implementation of the skeleton of a mobile version of a website. This request is intended to get approval on the 'mobile-version' branch which was based on the master branch initially.

### CODE REVIEW

File: homepage.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                </button>
+                <button class="button">
+                    <img src="images/Angelist.png" alt="Angelist">
+                </button>
+                <button class="button">
+                    <img src="images/Twitter.png" alt="Twitter">
+                </button>
+                <button class="button">
+                    <img src="images/Medium.png" alt="Medium">
+                </button>
```
Comment (Ismailco) [inline homepage.html:43]: - [ ] The image list should be wrapped in a `ul` tag, and each `img` should be inside `li`, also the images should be a link so you need to add an `a` tag inside the `li`

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            </div>
+            <div class="illustration">
+                <img src="images/Illustration.png" alt="Illustration">
+            </div>
+        </div>
+    </div>
```
Comment (Ismailco) [inline homepage.html:49]: - [ ] Kindly add the right background image for this section and stick to the design as much as possible including the fonts, sizes, and margins

|Your design|Figma design|
|-|-|
|Screenshot_20220629_205204|Screenshot_20220629_210335|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @DebbyMiressa,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @DebbyMiressa, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**